airgun/widgets.py

```diff
@@ -2034,6 +2034,41 @@ def read(self):
         return self.progress
 
 
+class ProgressBarPF4(ProgressBar):
```

Comment:
Change the class name to PF4ProgressBar, as we use PF4 as prefix all the time

Reply:
Ah yeah, I use that convention everywhere else. Good catch.